.github/ISSUE_TEMPLATE/default.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Obol Security Issue
+about: Report a security issue within Obol Labs
+title: ''
+labels: 'security'
+assignees: ''
+
+---
+
+# ARE YOU SUBMITTING A SECURITY ISSUE FOR OBOL LABS?
+Please submit all security issues via this Github form. We take security incidents seriously and will prioritize their resolution.
+
+**Why?**
+To ensure that all security issues are triaged effectively, addressed promptly and kept track of, we encourage the use of this Github form. This way, we can organize, track, and manage security incidents in a streamlined manner.
+
+**NOTE:**
+This process is regularly reviewed, and the form and/or system may change over time. If you need an immediate response or the security incident is of high severity, please use an appropriate escalation such as a direct message to the #security channel in Slack, an escalation to your team, or open an incident in PagerDuty.
+
+## **If urgent, here's how to reach out to us on Slack:**
+
+1. Open the Slack application.
+2. In the "Jump to..." search bar at the top, type `#security` and select the channel.
+3. In the message field at the bottom, describe the security issue you're facing. Please provide as much context as possible.
+4. Press Enter to send the message. A member of our security team will review and respond as soon as possible.
```

Comment:
This should relate to our public discord, as this will be for external people to reach out

Reply:
I want us to be able to use this internally as well, maybe I can add options for both

Reply:
Not sure about mixing internal and external, also not sure if anyone internally would start with going to obol-security with an urgent query. They'd just use slack in some form anyway

Reply:
The reason U'm creating this is because I get a lot of security related request from people that's not tracked and I want to create a process for people to create security task requests that are not urgent.

Also, didn't want to create to create another repository as I saw that this exists already